data/specials/hole.j3x

```diff
@@ -0,0 +1,19 @@
+string default_activation="OnEnter"
+list default_functions=["Teleport"]
+list default_datas=[(0,0)]
+
+vector size=(48.0,37.0)
```

Comment:
Previous size was (48.0, 48.0) - it was a square, but hole.png wasn't square at all. Size needs to be set to exact value of size in the texture. Also, textures needs to small - e.g. 48 x 37, not 800 x 600 because it is not efficient (it's pixelart game).